packages/db/src/translate/collections.ts

```diff
@@ -1,6 +1,37 @@
-import { NewCollectionRecord } from '@cumulus/types/api/collections';
-import { PostgresCollection } from '../types/collection';
+import { NewCollectionRecord, CollectionRecord } from '@cumulus/types/api/collections';
+import { PostgresCollection, PostgresCollectionRecord } from '../types/collection';
+const { removeNilProperties } = require('@cumulus/common/util');
 
+/**
+* Translates a PostgresCollectionRecord object to a `CollectionRecord` API collection object
+* @param {PostgresCollectionRecord} collectionRecord - PostgreSQL collection record to translate
+* @returns {CollectionRecord} - Translated record
+*/
+export const translatePostgresCollectionToApiCollection = (
+  collectionRecord: PostgresCollectionRecord
+): CollectionRecord => removeNilProperties(({
+  createdAt: collectionRecord.created_at.getTime(),
+  updatedAt: collectionRecord.updated_at.getTime(),
+  name: collectionRecord.name,
+  version: collectionRecord.version,
+  process: collectionRecord.process,
+  url_path: collectionRecord.url_path,
+  duplicateHandling: collectionRecord.duplicate_handling,
+  granuleId: collectionRecord.granule_id_validation_regex,
+  granuleIdExtraction: collectionRecord.granule_id_extraction_regex,
+  files: collectionRecord.files,
+  reportToEms: collectionRecord.report_to_ems,
+  sampleFileName: collectionRecord.sample_file_name,
+  ignoreFilesConfigForDiscovery: collectionRecord.ignore_files_config_for_discovery,
+  meta: collectionRecord.meta ?? undefined,
+  tags: collectionRecord.tags ?? undefined,
```

Comment:
Is the `??` operator necessary? if `collectionRecord.tags` is `undefined`, then it'll already be assigned as `undefined`?

Reply:
It isn't - functionally this only matters if you're being explicit that meta/tags should not be null, and that's entirely unneeded here. 

af0d0d1b5